steps/apk-info/1.2.3/step.yml

```diff
@@ -0,0 +1,51 @@
+title: APK info
+summary: APK Android info provider
+description: Provides all possible Android APK information as package name, version
+  name or version code.
+website: https://github.com/thefuntasty/bitrise-step-apk-info
+source_code_url: https://github.com/thefuntasty/bitrise-step-apk-info
+support_url: https://github.com/thefuntasty/bitrise-step-apk-info/issues
+published_at: 2017-06-27T08:58:54.106506707+02:00
+source:
+  git: https://github.com/thefuntasty/bitrise-step-apk-info.git
+  commit: b8763ea4d0517c0ec16ceca37cb62373c8ad833e
+project_type_tags:
+- android
+type_tags:
```

Comment:
please update the step's `type_tags` based on our [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention) and re-share your step